packages/form-control/src/use-form-control.ts

```diff
@@ -25,18 +25,11 @@ export function useFormControl<T extends HTMLElement>(
   const field = useFormControlContext()
   const describedBy: string[] = []
 
-  if (field?.isInvalid) {
-    /**
-     * Error message must be described first
-     * in all scenarios
-     */
-    if (describedBy.length > 0) {
-      describedBy.unshift(field.feedbackId)
-    } else {
-      describedBy.push(field.feedbackId)
-    }
-  }
-
+  /**
+   * Error message must be described first
+   * in all scenarios
+   */
+  if (field?.isInvalid) describedBy.push(field.feedbackId)
```

Comment:
Do we want to add a check similar to `hasHelpText ` here? Currently it always adds feedbackId when isInvalid, even though there might not be one.

Reply:
Yes, that make sense. Please add it